openml_OS/models/api/v1/Api_data.php

```diff
@@ -236,8 +241,34 @@ private function data_list($segs) {
 
     $this->xmlContents('data', $this->version, array('datasets' => $datasets));
   }
+  
+  private function data_fork() {
+    // get data id
+    $data_id = $this->input->post('data_id');
+    // If data id is not given
+    if( $data_id == false ) {
+      $this->returnError( 1070, $this->version );
```

Comment:
no spaces after ( or before ) (I know we are not consistent with this, but would be good to keep this in mind)